client/my-sites/plan-features/actions.jsx

```diff
@@ -85,11 +86,13 @@ const PlanFeaturesActions = ( {
 				} );
 			}
 		} else if ( isInSignup ) {
-			buttonText = translate( 'Start with %(plan)s', {
-				args: {
-					plan: planName,
-				},
-			} );
+			buttonText = isEligibleForPlanStepTest
+				? 'Get Started'
```

Comment:
All the other texts looks like they're sentence case, e.g., `Get started`

Reply:
I checked past testing P2s and saw that we've previously used camel case as CTA text in some cases, for example in p8Eqe3-AU-p2. The sentence case is the recommended one per style guide though. I defer to @victorespigares and @donlair to take a call on whether the CTA text should change from `Get Started` to `Get started`.

Reply:
Thanks for reviewing this Ramon, and thanks for the ping Niranjan! I don’t have a strong feeling about this, so let’s go with “Get started” to keep the style consistent there.

Reply:
`Get Started` is good in camel for the CTAs.

Reply:
Fixed.